src/tool/cppwinrt/cppwinrt/main.cpp

```diff
@@ -198,6 +203,36 @@ Where <spec> is one or more of:
         settings.component_filter = { settings.include.empty() ? include : settings.include, settings.exclude };
     }
 
+    static void build_fastabi_cache(cache const& c)
+    {
+        if (!settings.fastabi)
+        {
+            return;
+        }
+
+        for (auto&& [ns, members] : c.namespaces())
+        {
+            for (auto&& type : members.classes)
+            {
+                if (!has_fastabi(type))
+                {
+                    continue;
+                }
+
+                auto default_interface = get_default_interface(type);
+
+                if (default_interface.type() == TypeDefOrRef::TypeDef)
```

Comment:
>default_interface [](start = 20, length = 17)

is it possible for a default interface to be generic?

Reply:
Yes, although then it would not be exclusive and thus not eligible for the fast ABI.